frame/examples/dev-mode/src/lib.rs

```diff
@@ -78,7 +78,7 @@ pub mod pallet {
 			Ok(())
 		}
 
-		#[pallet::call_index(1)]
+		/// No need to define a `call_index` attribute here because of `dev_mode`.
 		/// No need to define a `weight` attribute here because of `dev_mode`.
```

Comment:
```suggestion
		// No need to define a `call_index` attribute here because of `dev_mode`.
		// No need to define a `weight` attribute here because of `dev_mode`.
```